src/common/tusb_debug.h

```diff
@@ -43,7 +43,7 @@
 #if CFG_TUSB_DEBUG
 
 // Enum to String for debugging purposes
-#if CFG_TUSB_DEBUG >= 2
+#if CFG_TUSB_DEBUG >= CFG_TUH_LOG_LEVEL
```

Comment:
since it is common for host and device, it should be

```C
#if CFG_TUSB_DEBUG >= CFG_TUH_LOG_LEVEL || CFG_TUSB_DEBUG >= CFG_TUD_LOG_LEVEL
```

Reply:
ok